asb-ballerina/src/asb/sender_connection.bal

```diff
@@ -0,0 +1,112 @@
+    public isolated function init(ConnectionConfiguration connectionConfiguration) {
+        self.connectionString = connectionConfiguration.connectionString;
+        self.entityPath = connectionConfiguration.entityPath;
+        self.asbSenderConnection = <handle> createSenderConnection(java:fromString(self.connectionString),
+            java:fromString(self.entityPath));
+    }
+
+    # Creates a Asb Sender Connection using the given connection parameters.
+    # 
+    # + connectionConfiguration - Configurations used to create a `asb:Connection`
+    # + return - An `asb:Error` if failed to create connection or else `()`
+    public isolated function createSenderConnection(ConnectionConfiguration connectionConfiguration) 
+        returns handle|Error? {
+        self.connectionString = connectionConfiguration.connectionString;
+        self.entityPath = connectionConfiguration.entityPath;
+        self.asbSenderConnection = <handle> createSenderConnection(java:fromString(self.connectionString),
+            java:fromString(self.entityPath));
+    }
+
+    # Closes the Asb Sender Connection using the given connection parameters.
+    #
+    # + return - An `asb:Error` if failed to close connection or else `()`
+    public isolated function closeSenderConnection() returns Error? {
+        return closeSenderConnection(self.asbSenderConnection);
+    }
+
+    # Send message to queue with a content and optional parameters
+    #
+    # + content - MessageBody content
+    # + parameters - Optional Message parameters 
+    # + properties - Message properties
+    # + return - An `asb:Error` if failed to send message or else `()`
+    public isolated function sendMessageWithConfigurableParameters(byte[] content, map<string> parameters,
+        map<string> properties) returns Error? {
+        return sendMessageWithConfigurableParameters(self.asbSenderConnection, content, parameters, properties);
+    }
+
+    # Send message to queue with a content
+    #
+    # + content - MessageBody content
+    # + contentType - Content type of the message content
+    # + messageId - This is a user-defined value that Service Bus can use to identify duplicate messages, if enabled
+    # + to - Send to address
+    # + replyTo - Address of the queue to reply to
+    # + label - Application specific label
+    # + sessionId - Identifier of the session
+    # + correlationId - Identifier of the correlation
+    # + properties - Message properties
+    # + timeToLive - This is the duration, in ticks, that a message is valid. The duration starts from when the 
+    #                message is sent to the Service Bus.
+    # + return - An `asb:Error` if failed to send message or else `()`
+    public isolated function sendMessage(byte[] content, string contentType, string messageId, string to, 
+        string replyTo, string label, string sessionId, string correlationId, map<string> properties, int timeToLive) 
```

Comment:
Can move some part of this line to the previous line.

Reply:
It exceeds